src/knownhost.c

```diff
@@ -1017,6 +1021,14 @@ knownhost_writeline(LIBSSH2_KNOWNHOSTS *hosts,
         key_type_name = "ssh-dss";
         key_type_len = 7;
         break;
+    case LIBSSH2_KNOWNHOST_KEY_ECDSA:
+        key_type_name = "ecdsa-sha2-nistp256";
```

Comment:
If we support other key lengths, we can't assume ecdsa-sha2-nistp256 here. Should we check key length to determine name (ecdsa-sha2-nistp256, ecdsa-sha2-nistp384, ecdsa-sha2-nistp521)?